Formula/bookloupe.rb

```diff
@@ -0,0 +1,62 @@
+class Bookloupe < Formula
+  desc "Lists common formatting errors in a Project Gutenberg candidate file"
+  homepage "http://www.juiblex.co.uk/pgdp/bookloupe/"
+  url "http://www.juiblex.co.uk/pgdp/bookloupe/bookloupe-2.0.tar.gz"
+  sha256 "15b1f5a0fa01e7c0a0752c282f8a354d3dc9edbefc677e6e42044771d5abe3c9"
+
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    ln_s("#{bin}/bookloupe", "./bookloupe")
```

Comment:
The tests failed if `bookloupe` was not present in the cwd. After reviewing the source code, I found this can be done with an environment variable, I will squash and re-push with that instead.